src/components/Task.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div id="task">
+   <h1>ToDo リスト</h1>
+
+    <input type="radio" name="condition" value="すべて" v-model="picked" checked>すべて
+    <input type="radio" name="condition" value="作業中" v-model="picked">作業中
+    <input type="radio" name="condition" value="完了" v-model="picked">完了
+
+    <table :class="table">
+      <thead>
+        <tr>
+          <th>ID</th>
+          <th>コメント</th>
+          <th>状態</th>
+        </tr>
+      </thead>
+      <tbody v-for="todos in filterStatus" :key="todos.id">
+        <tr>
+          <td>{{todos.id}}</td>
+          <td>{{todos.content}}</td>
+          <td>
+            <button @click="changeStatus(todos.id)">{{todos.status}}</button>
+            <button @click="removeTodo(todos.id)">削除</button>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    <h2>新規タスクの追加</h2>
+    <input type="text" name="newTask" v-model="text">
+    <button @click="addTodo">追加</button>
+  </div>
+</template>
+
+<script>
+export default{
+  data: function() {
+    return {
+      todo: {
+        id: '',
+        contnet: '',
+        status: ''
+      },
+      tasks: [],
+      picked: 'すべて',
+      text: '',
+      count: 0,
+      /*CSSクラス*/
+      table: 'table'
+    }
+  },
+  computed: {
+    filterStatus: function() {
+      let filterTasks = null;
+      if (this.picked === 'すべて') {
+        filterTasks = this.tasks;
+      } else if (this.picked === '作業中') {
+        filterTasks = this.tasks.filter( function(list) {
+          return list.status === '作業中';
+        });

```

Comment:
filterした値をそのまま返却すれば、filterTasksの定義を削除できます。
また、filterのような配列のループ処理で第一引数に配列の中身が入る場合、命名は配列名の複数形を表す「s」を取った命名が自然です。
そして処理内容がreturn文一行なので、アロー関数も併用すると省略できる部分がでできます。
まとめますと以下のようになります。
```suggestion
        this.tasks.filter(task => task.status === '作業中');
```
他の箇所も修正お願いします。